utils/config/encryption.go

```diff
@@ -114,14 +95,21 @@ func handleSecrets(config *Config, handler secretHandler, key string) error {
 	return nil
 }
 
-func getMasterKeyFromSecurityConfFile() (key string, secFileExists bool, err error) {
+func getMasterKey() (string, error) {
+	if key, exist := os.LookupEnv(coreutils.EncryptionKey); exist {
+		return key, nil
+	}
+	return getMasterKeyFromSecurityConfFile()
+}
+
+func getMasterKeyFromSecurityConfFile() (key string, err error) {
```

Comment:
getMasterKeyFromSecurityConfFile --> getEncryptionKeyFromSecurityConfFile
(for naming consistency reasons)